.github/workflows/msbuild.yml

```diff
@@ -1,14 +1,21 @@
 name: Build and Test
 
 on:
+  workflow_dispatch:
   pull_request:
     branches: [ master ]
+    paths-ignore:
+      - 'docs/**'
```

Comment:
I fear that my suggestion wasn't though out after all. The branch protection rule forcing success on this workflow might stand in the way of merging doc changes to master. I set up an experiment [here](https://github.com/dafny-lang/dafny/pull/3526).